src/components/pages/weather/elements/WeatherErrorMessage.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+
+export default class WeatherErrorMessage extends Component {
+  render() {
+    return (
+      <div className="pressure-cell">
```

Comment:
It looks like we're pulling the className up to the WeatherCell component, so we probably don't need this?